internal/printer/printer_css_test.go

```diff
@@ -0,0 +1,88 @@
+package printer
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+
+	astro "github.com/withastro/compiler/internal"
+	"github.com/withastro/compiler/internal/test_utils"
+	"github.com/withastro/compiler/internal/transform"
+)
+
+func TestPrinterCSS(t *testing.T) {
+	longRandomString := ""
+	for i := 0; i < 4080; i++ {
+		longRandomString += string(NON_WHITESPACE_CHARS[rand.Intn(len(NON_WHITESPACE_CHARS))])
```

Comment:
You might not need this… this was to test that length bug where our parser only parsed `4080` characters at a time